packages/web-twig/src/Resources/components/Select/Select.stories.twig

```diff
@@ -0,0 +1,23 @@
+{% extends 'layout/plain.html.twig' %}
+
+{% block content %}
+
+    {# Select Default #}
+    {% include '@components/Select/stories/Select.twig' %}
```

Comment:
please run the demo and see yourself... This doesn't render nicely.

Reply:
fixed